strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -190,20 +191,16 @@ protected boolean isChecksum(String fileName)
         return result;
     }
 
-    public URL resolveResource(RepositoryPath repositoryPath)
+    public URL resolveResourceURL(RepositoryPath repositoryPath)
```

Comment:
Cosmetic: Would you mind renaming this to `resolveResourceUrl`? I'm all up for using uppercase for abbreviations like these, but, across the codebase, camelhump has become the standard for things like these (for example, `myDao`, `myDto`, etc). Thanks!